lms/templates/courseware/course_about.html

```diff
@@ -328,6 +329,13 @@ <h2 class="opencourseware">MITOpenCourseware</h2>
     </div>
       %endif
 
+    % if waffle().is_enabled(ENABLE_COURSE_ABOUT_SIDEBAR_HTML):
+      % if get_course_about_section(request, course, "about_sidebar_html"):
```

Comment:
this only checks for content if the waffle switch is enabled, so the switch acts like a ripcord if bad content has been added.